empire

```diff
@@ -110,80 +110,7 @@ def execute_db_query(conn, query, args=None):
     cur.close()
     return results
 
-
```

Comment:
We don't want to remove the documentation at this time. But the other changes make sense to me.
Thanks for updating this.

Reply:
Done! :)